wagtail/admin/templates/wagtailadmin/shared/side_panel_toggle.html

```diff
@@ -3,7 +3,7 @@
 <button type="button"
     class="{{ nav_icon_button_classes }}"
     aria-label="{{ toggle.aria_label }}"
-    data-tippy-content="{{ panel.title }}"
+    data-controller="{{ panel.title }}"
     data-tippy-offset="[0,0]"
     data-tippy-placement="bottom"
```

Comment:
Sorry, this will not work see main feedback, 

data-controller should be w-tooltip
content/offset need to be replaced with their Stimulus variants e.g. data-w-tooltip-offset-value/data-w-tooltip-offset-value.

Finally, data-tippy-placement can be removed here as the new default is the bottom.